scenes/game/local_game.gd

```diff
@@ -1826,24 +1911,39 @@ class Player:
 					"attack_is_ex":
 						strike_stat_boosts.remove_ex()
 					"dodge_at_range":
-						var dodge_range = str(strike_stat_boosts.dodge_at_range_min)
-						if strike_stat_boosts.dodge_at_range_min != strike_stat_boosts.dodge_at_range_max:
-							dodge_range += "-%s" % strike_stat_boosts.dodge_at_range_max
-						parent._append_log_full(Enums.LogType.LogType_Effect, self, "will no longer dodge attacks from range %s." % dodge_range)
-						strike_stat_boosts.dodge_at_range_min = -1
-						strike_stat_boosts.dodge_at_range_max = -1
-						strike_stat_boosts.dodge_at_range_from_buddy = false
+						if 'special_range' in effect:
```

Comment:
probably should check the value of the special range just for future clarity

Reply:
just that it is set to something non- ""?
I can add that.